src/helpers/messages/remove_reaction.ts

```diff
@@ -1,12 +1,18 @@
 import { rest } from "../../rest/rest.ts";
 import { endpoints } from "../../util/constants.ts";
+import { requireBotChannelPermissions } from "../../util/permissions.ts";
 
-/** Removes a reaction from the bot on this message. Reaction takes the form of **name:id** for custom guild emoji, or Unicode characters. */
+/** Removes a reaction from the given user on this message, defaults to bot. Reaction takes the form of **name:id** for custom guild emoji, or Unicode characters. */
 export async function removeReaction(
   channelId: string,
   messageId: string,
   reaction: string,
+  userId?: string,
```

Comment:
Make this into an object so in future we can add more optional things and not have a breaking change

Reply:
{ userId?: string }